src/plugins/downloader/templates/download-manager.tsx

```
@@ -0,0 +1,479 @@
+export function DownloadManagerPanel(props: DownloadManagerProps) {
+  const [isOpen, setIsOpen] = createSignal(false);
+  const [activeTab, setActiveTab] = createSignal<TabType>('queue');
+  const [items, setItems] = createSignal<DownloadItem[]>([]);
+  const [maxConcurrent, setMaxConcurrent] = createSignal(1);
+  const [isPaused, setIsPaused] = createSignal(false);
+  const [totalCompleted, setTotalCompleted] = createSignal(0);
+  const [totalFailed, setTotalFailed] = createSignal(0);
+  const [totalSkipped, setTotalSkipped] = createSignal(0);
+
+  // Draggable window state
+  const [position, setPosition] = createSignal({ x: window.innerWidth - 420, y: 50 });
```

ESLint

[error] Replace `·x:·window.innerWidth·-·420,·y:·50` with `⏎····x:·window.innerWidth·-·420,⏎····y:·50,⏎·`

(prettier/prettier)